src/main/scala/org/bitcoins/core/script/crypto/HashType.scala

```diff
@@ -1,13 +1,14 @@
 package org.bitcoins.core.script.crypto
 
+import org.bitcoins.core.number.Int32
 import org.bitcoins.core.util.{BitcoinSUtil}
 
 /**
  * Created by chris on 1/18/16.
  */
 sealed trait HashType {
-  def byte : Byte
-  def hex : String = BitcoinSUtil.encodeHex(byte)
+  def hashType : Int32
```

Comment:
I think we should follow our naming convention of using `underlying` to represent the underlying number